docs/idm-concept.md

```diff
@@ -0,0 +1,206 @@
+
+A DID always identifies a person, organization or thing. They have a specific syntax: `<scheme>:<method>:<method-identifier>` (e.g. `did:ipid:a1B2c3d4E5`).
+
+DIDs resolve to DID-Documents — simple documents that describe how to use that specific DID. Each DID-Document contains at least three things: Cryptographic Material, Authentication Suites and Service Endpoints. Cryptographic Material combined with Authentication Suites provide a set of mechanisms to authenticate as the DID subject (e.g. public keys, pseudonymous biometric protocols, etc.). Service Endpoints enable trusted interactions with the DID subject.
+
+In the real world, people may use different devices to interact with others. Each of those should have their public keys listed in the DID-Document, making it possible to add and remove them as needed while keeping the same DID.
+
+#### Granular permissions & authentication
+
+As previously stated, people use different devices in their daily lives. Each of those devices have different security guarantees. As an example, a device in your home is less likely to be lost or stolen when compared to a mobile phone. On the other hand, a mobile phone may have its storage encrypted which is more secure than an unencrypted one. Having that said, the level of permissions should be considered when adding a new device.
+
+Moreover, if a device is going to be used for authentication, its public key should be listed in the `authentication` field of the DID-Document. This is necessary for `DID-Auth` or other authentication mechanisms to assert that the device is valid for authentication.
+
+### Verifiable Credentials
+
+Verifiable Credentials is a format for interoperable, cryptographically-verifiable digital credentials. DIDs begin by being "trustless" in the sense that they don't directly provide meaningful identity attributes. But trust between DID-identified peers can be built up through the exchange of verifiable credentials - credentials about identity attributes that include cryptographic proof. These proofs can be verified by reference to the issuer's DID and DID-Document.
+
+Because the ecosystem is still in its infancy and there's a lack of trusted issuers, identities may self-issue credentials. More specifically, they may issue credentials that define personal attributes about themselves, like their name and birthdate, and credentials that prove they own certain profiles on social networks, similar to how [Keybase](https://keybase.io) does. As of today, many people trust the mainstream social networks, such as Facebook and Twitter, and identities may use them to post cryptographic proofs that link their profiles to a hash of their DID. As time passes by and the ecosystem gets mature, identities will hold credentials issued by others that may be used in a variety of scenarios and purposes.

```

Comment:
to maintain formal registry, replace the 3 instances of _there's_ with _there is_

Reply:
Done!